README.md

```diff
@@ -42,4 +31,37 @@ to help out.
 Ryan T. Sammartino
 ryan.sammartino at gmail dot com
 
-$Id: README,v 1.13 2006/04/23 01:20:15 ryants Exp $
+## License
+
+Copyright (C) 2001-2010 Ryan T. Sammartino
+ryan.sammartino at gmail dot com
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2, or (at your option)
+any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+## Install
+
+Doxymacs depends on the following packages:
+
+- W3      http://www.cs.indiana.edu/usr/local/www/elisp/w3/docs.html
+- tempo   http://www.lysator.liu.se/~davidk/elisp/
+- libxml2 http://www.libxml.org/
+
+Be sure these are properly configured and installed before proceeding.
+
+- Use the configure script to configure doxymacs:
+
+   $ ./configure
+   $ make
+   $ make install
+
+  Use ./configure --help for help on customising your configuration.
+
+for details see [[INSTALL][INSTALL]]
```

Comment:
This looks like an org-mode link.  Could you change it to Markdown?  I'm wondering whether `[INSTALL](INSTALL)` will render correctly on the GitHub web interface, and this is a quick test with the Markdown change.

Reply:
Link changed to Markdown, works as expected. ;)
I experimented with org mode as this is mostly also rendered by GitHub.